websites/K/Kahoot/presence.ts

```diff
@@ -64,127 +87,323 @@ presence.on("UpdateData", async () => {
 	strings ??= await getStrings();
 	if (oldLang !== newLang) oldLang = newLang;
 
-	if (document.location.href.match("https://kahoot.it")) {
-		const path = document.location.pathname;
```

Comment:
You should keep this and destructure `document.location.pathname` and `document.location.host` for simplicity